src/prasadSDec2019/Assignment3/Question5.java

```diff
@@ -0,0 +1,29 @@
+package prasadSDec2019.Assignment3;
+
+import java.util.Scanner;
+
+public class Question5 {
+	
+	void reverseNum() {
+		int num;
+		System.out.println("Enter total numbr of elements");
+		Scanner scn=new Scanner(System.in);
+		num=scn.nextInt();
+		System.out.print("Inputs are:");
+		int a[]=new int[num];
+		for(int i=0;i<num;i++) {
```

Comment:
Print some statement so user have some idea to still he/she need to give input..